packaging/npm-publish.sh

```diff
@@ -39,7 +40,9 @@ $NIMIQ_VERSION
     "worker-js.js",
     "worker.js.map",
     "worker-wasm.js",
-    "worker-wasm.wasm"
+    "worker-wasm.wasm",
+    "types.d.ts",
+    "namespace.d.ts"
   ]
 }
 _EOF_
```

Comment:
I believe you need to add the files `types.d.ts` and `namespace.d.ts` also to the `cp ...` command in line 50 of this file to copy them from /dist to the /npm packaging directory.

Reply:
That's true